scripts/test/Muon/phase_table_widget/phase_table_presenter_test.py

```diff
@@ -217,24 +217,39 @@ def test_add_fitting_info_to_ADS_adds_fitting_info_to_ADS_if_option_selected(sel
         workspace_wrapper_mock.return_value.show.assert_called_once_with()
 
     def test_that_disable_observer_calls_on_view_when_triggered(self):
-        self.view.setEnabled(True)
-        self.assertTrue(self.view.isEnabled())
+        self.view.enable_widget()
+
+        for widget in self.view.children():
+            if str(widget.objectName()) in ['cancel_button', 'phasequad_cancel_button']:
+                continue
```

Comment:
unless I am misunderstanding this says that if it is the cancel button we dont do the assert. Is that what you meant to do?

Reply:
Yes as the enable_widget and disable_widget method does not affect the those buttons.